R/tranform_assay.R

```diff
@@ -134,6 +133,9 @@ transform_utility  = function(input_read_RNA_assay, transform_fx, external_path,
   # Apply the transformation method to counts
   counts <- transform_function(counts)
   
+  # Apply the transformation method to 5000 cells
+  counts_light_for_checks <- transform_function(counts_light_for_checks)
```

Comment:
would not be better to move 

`counts_light_for_checks = counts[,sample(seq_len(ncol(counts)), size = min(numer_of_cells_to_sample, ncol(counts))),drop=FALSE]`

after line 134. So we don't transform twice, but `counts_light_for_checks` are taken from the transformed `counts`?

also density est should then follow `counts_light_for_checks`